package.json

```
@@ -44,7 +44,9 @@
     "lint": "eslint src --ext .ts,.tsx",
```

Biome (2.4.7)

[error] The key lint was already declared.

(lint/suspicious/noDuplicateObjectKeys)